Examples/Sources/MacroExamples/Implementation/Peer/AddAsyncMacro.swift

```diff
@@ -58,7 +60,9 @@ public struct AddAsyncMacro: PeerMacro {
     }
 
     // Completion handler needs to return Void
-    if completionHandlerParameter.returnClause.type.with(\.leadingTrivia, []).with(\.trailingTrivia, []).description != "Void" {
+    guard let returnIdentifier = completionHandlerParameter.returnClause.type.as(IdentifierTypeSyntax.self),
+      returnIdentifier.name.text == "Void"
+    else {
```

Comment:
Similar here, any reason why you chose to store the `returnIdentifier` in a variable?